_posts/2019-3-29-Copr-new-rss-feed.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Copr has a new RSS feed
+author: Jan Buchmaier
+layout: post
+---
+
+Now you can keep track off of every new Copr thanks to our new feed.
+ 
+
+## What is the content of the feed
+
+Our RSS feed contains 200 of the late projects.
+
+And in every item you can find:
+
+ * author's and project's name
+ * link to the project 
+ * simple description
+
+
+## Where to find it
+
+You can visit our feed [here][link].
+
+[link]:https://copr.fedorainfracloud.org/rss/ "RSS feed"
```

Comment:
This is really cool, I had no idea, that this was possible to do in markdown.
Thank you!

Reply:
I try to avoid "here" word in links usually, so e.g. `You can visit [our feed][link]` would be slightly better, but it is not strict requirement :-)